digdag-standards/src/main/java/io/digdag/standards/operator/td/TDOperator.java

```diff
@@ -1,33 +1,43 @@
 package io.digdag.standards.operator.td;
 
-import java.util.Date;
-import java.io.Closeable;
-
+import com.google.common.base.Optional;
 import com.google.common.base.Throwables;
-import io.digdag.client.config.Config;
-import io.digdag.client.config.ConfigException;
-import io.digdag.util.RetryExecutor;
-import io.digdag.util.RetryExecutor.RetryGiveupException;
 import com.treasuredata.client.TDClient;
 import com.treasuredata.client.TDClientException;
 import com.treasuredata.client.TDClientHttpConflictException;
 import com.treasuredata.client.TDClientHttpNotFoundException;
 import com.treasuredata.client.TDClientHttpUnauthorizedException;
-import com.treasuredata.client.model.TDJobRequest;
 import com.treasuredata.client.model.TDExportJobRequest;
+import com.treasuredata.client.model.TDJobRequest;
+import io.digdag.client.config.Config;
+import io.digdag.client.config.ConfigException;
+import io.digdag.util.RetryExecutor;
+import io.digdag.util.RetryExecutor.RetryGiveupException;
+
+import java.io.Closeable;
+import java.util.Date;
+
 import static io.digdag.util.RetryExecutor.retryExecutor;
 
 public class TDOperator
         implements Closeable
 {
-    public static TDOperator fromConfig(Config config)
+    public static TDOperator fromConfig(Config config) {
+        return fromConfig(config, false);
+    }
+
+    public static TDOperator fromConfigWithRetries(Config config) {
```

Comment:
I think TDOperator should use either TDClient's retry or TDOperator's retry. Using both of them means that retrying in TDClient AND retrying in TDOperator. It may retry for unexpected amount of time.
Unless all TDClient API become ready to retry, we should use TDOperator's retry only and always disable TDClient's retry.